src/sm/sm_main/sm_main_exec.js

```diff
@@ -3101,8 +3074,9 @@ function eval_functionCall(ctx, tag) {
         return eval_ARITH_BN254_ADDFP2(ctx, tag);
     } else if (tag.funcName == "ARITH_BN254_SUBFP2") {
         return eval_ARITH_BN254_SUBFP2(ctx, tag);
+    } else {
+        return ctx.helpers.Rom.evalRomCommand(ctx, tag)
```

Comment:
this call must be removed, ctx.helpers.Rom.evalRomCommand(ctx, tag) because ROM helpers was called from line 3000, const res = ctx.helpers.callDefault('eval_'+ tag.funcName, [ctx, tag], null);